docs/docs/testkube-cloud/articles/status-pages.md

```diff
@@ -0,0 +1,238 @@
+
+## Overview
+
+![status-page-main](../../img/status-page-main.png)
+
+Testkube Status Pages is a feature within the Testkube software that allows you to create and manage dedicated status pages for your software projects. These pages provide real-time information about the health and availability of your project components, making it easier for both internal teams and external stakeholders to stay informed.
+
+### Key Benefits
+
+* Transparency: Keep all stakeholders informed about the status of your software.
+* Efficiency: Quickly detect and respond to issues, minimizing downtime.
+* Customization: Tailor your status page to match your brand and information needs.
+
+## Getting Started
+
+To access Testkube Status Pages:
+
+1. Log in to your Testkube Cloud account.
+2. Go to the dashboard.
+3. Click on "Status Pages" in the left navigation panel.
+
+### Creating Your First Status Page
+
+![status-page-edit](../../img/status-page-edit.png)
+
+1. Provide a unique name and description for your status page.
+2. Choose a time scale for the status page. This will define the aggregation period of the test execution results.
+3. Add services with tests.
+4. Configure visibility. Public status pages will be available without any kind of authentication.
+5. Save changes by clicking the "Save" button on top.
+
+## Managing Services
+
+Services represent the different parts of your software project that you want to monitor, such as servers or databases. In Testkube, you can define them by adding the appropriate tests to one Service in your Status Page.
+
+![status-page-edit-services](../../img/status-page-edit-services.png)
+
+To add and configure a service:
+
+1. In the services section of the status page, type the chosen name of your service into the "Add a service" field.
+2. Choose the relevant tests from the dropdown.
+3. Drag and drop to order them.
+
+## Visualising Service Statuses
+
+The Status Page rendered is unique for each Testkube environment. You can only have one Status Page per environment, so make sure the tests added offer an accurate representation of the status of the services. On the very top of the screen you will see the name of your Status Page and the given description. No details of your Testkube environment or organization will be published on this page, so this is where they need to be pointed out for efficient communication with your users.
+
+![status-page-service](../../img/status-page-service.png)
+
+Below that you will see a list of the services with multiple ways to represent their state. On the top left is the name of the service. Top right is current status - this is calculated based on the last bar of the chart below. The possible values are:
```

Comment:
```suggestion
Below that you will see a list of the services with multiple ways to represent their state. On the top left is the name of the service. Top right is current status - this is calculated based on the last bar of the chart below **(should this be 'above'?)**. The possible values are:
```

Reply:
It is referring to the chart below the current status, like in the `status-page-service.png` image.